app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -123,6 +129,24 @@ def process_collection(collection, key)
         end
       end
 
+      def gather_data_for_this_region(arm_service)
+        results = []
+        @data[:resource_groups].each do |resource_group|
+          results << arm_service.list(resource_group[:name])
```

Comment:
@djberg96 - assuming you mean:
   ` res = Azure::Armrest::ResourceService.new(c)`

This gives me back limited data:
    Azure::Armrest::Resource id=\"/subscriptions/462f2af8-e67e-40c6-9fbf-02824d1dd485/resourceGroups/ComputeVMs/providers/Microsoft.Compute/virtualMachines/chefserver1\", name=\"chefserver1\", type=\"Microsoft.Compute/virtualMachines\", location=\"eastus\", plan=<#<Class:0x007ff192c1b480> name=\"chefbyol\", product=\"chef-server\", publisher=\"chef-software\">

Whereas the VirtualMachineService gives me the data that I need:
https://gist.githubusercontent.com/bronaghs/13e0d0827b2fba8d6f44/raw/df3d9a7c0f021d43f44bba07eab185997414009a/gistfile1.txt

Am I missing something? Or can the ResourceService be updated to give me back a full picture of each inventory type without a lot of effort?